server/api.js

```diff
@@ -3,8 +3,9 @@ import db from "./db.js";
 const router = Router();
 
 router.get("/videos", async (_, res) => {
-	const result = await db.query("SELECT 'ok' ok");
-	res.json(result.rows[0]);
+	const result = await db.query("SELECT * FROM videos;");
```

Comment:
similarly, this is the solution for the first exercise, so not sure if we want to add it to the base project